applications/nrf5340_audio/src/bluetooth/le_audio_cis_headset.c

```diff
@@ -461,6 +461,7 @@ int le_audio_enable(le_audio_receive_cb recv_cb)
 {
 	int ret;
 
+	bt_conn_cb_register(&conn_callbacks);
```

Comment:
Same as above, we might need to move this line into initialize()

Reply:
Move to initialize().